app/rooms/commands/lobby-commands.ts

```diff
@@ -168,6 +168,85 @@ export class OnJoinCommand extends Command<
   }
 }
 
+export class LoadMoreHistoryCommand extends Command<
+  CustomLobbyRoom,
+  { client: Client; searchedUid: string | undefined; skip: number; limit: number }
+> {
+  async execute({ client, searchedUid, skip, limit }: { client: Client; searchedUid:string | undefined; skip: number; limit: number }) {
+    try {
+      // if searchedUid is defined return a new searched user, else update self user history.
+      if(searchedUid != undefined){
+        // console.log("load more history for searchedUid: ", searchedUid)
+        const user = await UserMetadata.findOne({ uid: searchedUid })
+        if (user) {
+          const stats = await DetailledStatistic.find(
+            { playerId: user.uid },
+            ["pokemons", "time", "rank", "elo"],
+            { limit: skip+limit, sort: { time: -1 } }
+          )
+          if (stats) {
+            client.send(
```

Comment:
sending the entire user won't be necessary once we'll have a dedicated http endpoint for game history